docker/pypi/wmagent/init.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+### Basic initialization wrapper for WMAgent to serve as the main entry point for the WMAgent Docker container
+
+
+echo "Start initialization"
+./run.sh || true
+
+echo "Start sleeping now ...zzz..."
+
+while true; do sleep 10; done
```

Comment:
Is this line just so the container keeps running? If that is the case, is there any reason to keep the start-agent line below?

Reply:
@amaltaro , I've always been with the impression the names of those two scripts should be swapped. And now as I look at the ASCII diagram showing script relations, this impression only solidifies. But I did not proceed with the renaming because this would smear all the changes and the Review would become practically impossible... I may suggest, once we are ready for merging as a last step I should swap the names of those two scripts in a single commit separate from the rest. What do you think about it ?

Reply:
@amaltaro , Sorry if it was not clear enough - I was talking about swapping the names of: `init.sh` <--> `run.sh` .
Such that `run.sh` gets back as a default entry point for Docker and just calls `init.sh` and goes into an endless loop, while `init.sh` drives the whole initialization process.

Reply:
@todor-ivanov apologies for the delay. You have a better view of this than me, so please go ahead if you consider it better. If you do so though, please make sure the README file reflects it as well.